bindata/v4.1.0/alerts/audit-errors.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: audit-errors
+  namespace: openshift-kube-apiserver
+spec:
+  groups:
+  - name: apiserver-audit
+    rules:
+    - alert: AuditLogError
+      annotations:
+        summary: |-
+          Incident response teams rely on the audit logs to detect malicious activity.
+          If an API Server is unable to succesfully write an event to an audit log,
+          this might hinder the incident response team in dealing with a security
+          incident.
+        description: An API Server had an error writing to an audit log.
+      expr: |
+        sum by (apiserver,instance)(rate(apiserver_audit_error_total{apiserver=~".+-apiserver"}[5m])) / sum by (apiserver,instance) (rate(apiserver_audit_event_total{apiserver=~".+-apiserver"}[5m])) > bool 0
+      for: 1m
+      labels:
+        severity: critical
```

Comment:
A critical alert will page cluster-owners, like SRE-P for example in the middle of the night for this. I would encourage you to drop down to a warning, as the clusters availability is not endangered in any way from this.

Reply:
changed it to a `warning`